manager/integration/tests/longhorn.py

```diff
@@ -172,12 +172,11 @@ def __repr(self):
 
 
 class ApiError(Exception):
-    def __init__(self, obj):
-        self.error = obj
+    def __init__(self, response):
+        self.status = response.status_code
         try:
-            msg = '{} : {}\n{}'.format(obj.code, obj.message, obj)
-            super(ApiError, self).__init__(self, msg)
-        except:
+            super(ApiError, self).__init__(self, response.text)
```

Comment:
It's better to preserve the format, since https://github.com/rancher/longhorn-manager/blob/master/api/router.go#L37 is still using it.

Reply:
you mean `self.error`?